### DISCUSSION TITLE
How to donate to Taskwarrior?

### DISCUSSION THREAD

**Discussion by FreddieGilbraith** [GothenburgBitFactory/taskwarrior#2486] [Q&A] [answered]
Hey,

I've used taskwarrior on and off for the last few years, and decided I should probably contribute something back. I'm no good with C, but would love to donate some money rather than time.

Is the paypal link on taskwarrior.org/ still correct and going to someone related to the project? Thought it was best to check before I donated.

Keep up the great work 👍 

(ps, if you setup an open-collective or patreon, I'd definitly support there too :))

**Comment by tbabej** [comment, marked as answer]
Hi @FreddieGilbraith, thanks for the kind words, it's much appreciated. We have actually recently setup a sponsors profile here on Github, which we haven't publicized yet (as we've been only recently approved).

If you'd consider becoming one of our early sponsors, please visit: https://github.com/sponsors/GothenburgBitFactory/

**Reply by FreddieGilbraith** [reply]
Done and done. Thanks very much for all your hard work

**Comment by MarkLeakos** [comment]
Been using taskwarrior for a while and recently became a sponsor. I was wondering how to access the indicated tips and videos that were stated for donors. Thanks.

**Reply by tbabej** [reply]
Hi Mark, the videos mentioned in the tiers are still work in progress. Once produced, we will share them with all current (and past) project sponsors using the profile email. Github now allows sharing the contact details with the sponsored org for such purposes, and the users that don't do so (and don't have a profile email) will be tagged in an announcement thread :slightly_smiling_face:

**Comment by MarkLeakos** [comment]
Thank you Tomas and keep up the great work. I really like TaskWarrior.

Sincerely,
Mark Leakos
